sys/include/vfs.h

```diff
@@ -82,11 +85,11 @@ extern "C" {
 #define _MAX(a, b) ((a) > (b) ? (a) : (b))
 #endif
 
-#ifndef MAX5
+#ifndef MAX6
 /**
  * @brief   MAX5 Function to get the largest of 5 values
  */
-#define MAX5(a, b, c, d, e) _MAX(_MAX(_MAX((a), (b)), _MAX((c),(d))), (e))
+#define MAX6(a, b, c, d, e, f) _MAX(_MAX(_MAX(_MAX((a), (b)), _MAX((c),(d))), (e)), (f))
```

Comment:
```suggestion
#define MAX6(a, b, c, d, e, f) MAX(MAX(MAX(MAX((a), (b)), MAX((c), (d))), (e)), (f))
```

There's a `MAX()` in `#include "macros/utils.h"` these days that can be used. Also: The comment above needs an update.